### PR TITLE
Added pagination and searching to the teams index

### PR DESCRIPTION
**Pull request by keithpitt**
screen shot 2017-04-06 at 3 34 26 pm
screen shot 2017-04-06 at 3 34 20 pm

### CODE REVIEW

File: app/components/team/Index.js [hunk 1]
```diff
@@ -61,19 +124,117 @@ class TeamIndex extends React.PureComponent {
...
   }
 
+  renderTeamSearchInfo() {
+    const { organization: { teams }, relay: { variables: { teamSearch } } } = this.props;
+
+    if (teamSearch && teams) {
+      return (
+        <div className="bg-silver semi-bold py2 px3">
+          <small className="dark-gray">
+            {formatNumber(teams.count)} matching teams
```
Comment (ticky) [inline app/components/team/Index.js]: ooh, it occurs to me we haven’t done pluralisation here (or in the Member list! (or the Agent list!))

Comment (keithpitt) [inline app/components/team/Index.js]: Oh yeah! Do we have pluralisation technology in `frontend`?

Comment (ticky) [inline app/components/team/Index.js]: We have only the mystic _ternary operator_, unfortunately.

Comment (ticky) [inline app/components/team/Index.js]: as the sages say,

> if the thing is not 1
> the thing has an s

Comment (keithpitt) [inline app/components/team/Index.js]: Such sage advice

**Pull request merged**